src/Linux/R/Interpreters/Impl/RInstallation.cs

```diff
@@ -36,9 +36,9 @@ public sealed class RInstallation : IRInstallationService {
             }
 
             if(package != null) {
-                return new RInterpreterInfo(name, path, package.Version, package.GetVersion(), _fileSystem);
+                return new RInterpreterInfo(name, package, package.Version, package.GetVersion(), _fileSystem);
             }
-            return new RInterpreterInfo(name, path, string.Empty, null, _fileSystem);
+            return new RInterpreterInfo(name, InstalledPackageInfo.EmptyPackage, string.Empty, null, _fileSystem);
```

Comment:
Given that the constructor extracts all paths from the package now, is this a valid case? i.e. would it be a usable interpreter if we couldn't find the package?